app/views/course/assessment/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.table-middle-align.table-hover
+  thead
+    tr
+      th = Course::Assessment::Submission.human_attribute_name(:course_user)
+      th = Course::Assessment::Submission.human_attribute_name(:assessment_title)
```

Comment:
`:assessment_title` is not a attribute in submission model, don't think we should use human attribute in this case.